ts/message/index.ts

```diff
@@ -0,0 +1,67 @@
+import rpcProtoStr from './rpc.proto'
+import protons = require('protons')
+
+const rpcProto = protons(rpcProtoStr)
+
+export interface SubOpts {
+  subscribe?: boolean
+  topicID?: string
+}
+
+export interface Message {
+  from?: Buffer
+  data?: Buffer
+  seqno?: Buffer
+  topicIDs: string[]
+  signature?: Buffer
+  key?: Buffer
+}
```

Comment:
Are all those really optional? you can use "Partial<Message>" if you need to initialize partially

Reply:
Yeah, they're optional. Its following the protobuf definition, we should call that out explicitly in the comments